snap/local/helper-go/go.mod

```
@@ -1,5 +1,5 @@
 module github.com/edgexfoundry/app-rfid-llrp-inventory/local/helper-go
 
-go 1.18
+go 1.208
```

I think this was a typo
```suggestion
go 1.20
```

---

Goo catch!

---

fix pushed